reg_tests/test_files/BLTFlatPlateT3A/BLTFlatPlateT3A.norm.gold

```diff
@@ -0,0 +1,5 @@
+137372539.0984875 1 0.01
```

Comment:
I think we typically let Jon generate the golds for the machines the tests are run on. Not super important.

Reply:
Yeah this is fine. @jrood-nrel will just rebless it on the test machine.